app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+
+from app.errors import NotVaccinatedError
+from app.errors import NotWearingMaskError
+from app.errors import OutdatedVaccineError
```

Comment:
```
from app.errors import (
    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```

Reply:
sorry, maybe I wasn't clear. That is what I meant

Reply:
oh, okay, i'll fix that